packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -203,14 +228,57 @@ class PageTransitionSwitcher extends StatefulWidget {
   /// The child provided to the transitionBuilder may be null.
   final PageTransitionSwitcherTransitionBuilder transitionBuilder;
 
+  /// A function that wraps all of the children that are transitioning out, and
+  /// the [child] that's transitioning in, with a widget that lays all of them
+  /// out. This is called every time this widget is built. The function must not
+  /// return null.
+  ///
+  /// The default is [PageTransitionSwitcher.defaultLayoutBuilder].
+  ///
+  /// See also:
+  ///
+  ///  * [PageTransitionSwitcherLayoutBuilder] for more information about
+  ///    how a layout builder should function.
+  ///
+  /// The following example shows a [layoutBuilder] that places the new child inside
+  /// a [Column], that sizes its height depending on the heights of active entries.
+  ///
+  /// ```dart
+  /// layoutBuilder: (
+  ///   List<ChildEntry> activeEntries,
+  /// ) => Column(
+  ///   children: activeEntries
+  ///     .map<Widget>((ChildEntry entry) => entry.transition)
+  ///     .toList(),
+  /// ),
+  /// ```
+  ///
+  final PageTransitionSwitcherLayoutBuilder layoutBuilder;
+
+  /// The layout builder used as the default value of [layoutBuilder].
+  ///
+  /// The new child is placed in a [Stack] that sizes itself to match the
+  /// largest of the child or a previous child. The children are centered on
+  /// each other.
```

Comment:
Also, doesn't the [Stack] size itself to match the largest child, not the previous child?

```suggestion
  /// This function is the default way for how the new and old child widgets are placed
  /// during the transition between the two widgets. The new child is placed in a 
  /// [Stack] that sizes itself to match the largest of the child or a previous child. 
  /// The children are centered on each other.
```